gradio/components/code.py

```diff
@@ -122,6 +122,7 @@ def __init__(
             elem_classes: An optional list of strings that are assigned as the classes of this component in the HTML DOM. Can be used for targeting CSS styles.
             render: If False, component will not render be rendered in the Blocks context. Should be used if the intention is to assign event listeners now but render the component later.
             key: if assigned, will be used to assume identity across a re-render. Components that have the same key across a re-render will have their value preserved.
+            lines: Maximum number of visible lines to show in the code editor. Overflow will be scrollable.
```

Comment:
Just tested and it looks like the `lines` parameter actually behaves differently than this. Its more like the `lines` parameter in `gr.Textbox`, which is the *minimum* number of visible lines to show. If the code takes up more than this number of lines, the full code will be shown. We *could* add a `max_lines` parameter, but that's a separate issues